packages/prettyprint/test/autotest/placeholders-string-double-backslash/expected.marko

```diff
@@ -1,3 +1,3 @@
-<div data-foo="\\${foo} \\$!{foo}"/>
+<div data-foo="\\${foo} \\${foo}"/>
 ~~~~~~~
-div data-foo="\\${foo} \\$!{foo}"
\ No newline at end of file
+div data-foo="\\${foo} \\${foo}"
```

Comment:
We probably shouldn't be removing the `!` here right?

Reply:
The behavior is the same between `${` and `$!{` in a string.  And there's nothing in the ast to differentiate them in this case.  We could change that, but I'm not sure there's a reason to.

Reply:
Right right. Yeah I think that's fine here.